go/apps/demo/demo_client/demo_client.go

```diff
@@ -126,6 +126,7 @@ func doClient(domain *tao.Domain) {
 			pingFail++
 		}
 		glog.Infof("client: made %d connections, finished %d ok, %d bad pings\n", i+1, pingGood, pingFail)
+		fmt.Printf("client: made %d connections, finished %d ok, %d bad pings\n", i+1, pingGood, pingFail)
```

Comment:
I assume the mix of fmt and glog here is to test the redirection. The glog library seems to write its logs to stderr (when it outputs them to the console), so that should all work correctly under redirected output.